packages/menu/src/Menu/Menu.types.ts

```diff
@@ -45,4 +45,9 @@ export interface MenuProps extends Omit<PopoverProps, 'active'> {
    * default: `false`
    */
   darkMode?: boolean;
+
+  /**
+   * id passed to the menu dropdown.
```

Comment:
where are most props spread?

Reply:
i'll keep the id with `...rest` so it spreads on the `ul`